install.py

```diff
@@ -1944,14 +2113,14 @@ def mkdir_p(cls, path):
 class Utils(object):
     """A general utility class."""
 
-    @classmethod
-    def version_str2tuple(cls, version_str):
-        """Version info.
+    @staticmethod
```

Comment:
Is `@staticmethod` better than `@classmethod`? Why do you prefer it?
I am not familiar with a recommended Python style. I just found below article about it.
https://stackoverflow.com/questions/136097/what-is-the-difference-between-staticmethod-and-classmethod#1669524

Reply:
Not necessarily "better", I just find it more suitable here. `@classmethod` is used to mark methods which just need access to the class' type but not to the specific instance. But this method doesn't even require that, in fact the `Utils` class is "just a namespace" (because everything has to be in a single file). So using `@staticmethod` appeared to me better here, as these are just free functions that are grouped inside a class.

Reply:
Alright. I respect your opinion. :)